packages/clickable/tests/clickable.test.tsx

```diff
@@ -1,90 +1,114 @@
-import { shallow } from "enzyme";
 import React from "react";
 import Clickable from "../components/clickable";
+import { render, fireEvent } from "@testing-library/react";
 
 describe("Clickable", () => {
   it("has role attribute", () => {
-    const wrapper = shallow(
+    const { getByRole } = render(
       <Clickable action={jest.fn()}>
         <span>onClick</span>
       </Clickable>
     );
-    expect(wrapper.prop("role")).toBe("button");
+
+    const buttonElement = getByRole("button");
+
+    expect(buttonElement).toBeInTheDocument();
```

Comment:
getByRole("button") is sufficient here. 

react testing library fails the test case if get element is not in DOM. So, checking if that element is in the document is redundant.